datacube/storage/netcdf_writer.py

```diff
@@ -154,10 +154,24 @@ def _write_transverse_mercator_params(crs_var, crs):
     crs_var.latitude_of_projection_origin = crs.proj.latitude_of_origin
 
 
+def _write_lcc2_params(crs_var, crs):
+    # e.g. http://spatialreference.org/ref/sr-org/mexico-inegi-lambert-conformal-conic/
+    crs_var.grid_mapping_name = 'lambert_conformal_conic_2sp'
```

Comment:
The [NetCDF CF Conventions](http://cfconventions.org/) for [Lambert conformal](http://cfconventions.org/cf-conventions/cf-conventions.html#_lambert_conformal) give the `grid_mapping_name` as simply `lambert_conformal_conic`, the `2sp` suffix in the WKT representation shouldn't be included.

Reply:
Great! I fixed it. And thanks for the netcdf conventions link